docs/modules/operation/pages/provisioning/introduction.adoc

```diff
@@ -26,45 +26,46 @@ Regardless of the method, provisioning is an iterative process: you will need to
 
 {page-component-title} uses an ICMP ping sweep to find IP addresses on the network and provision node entities.
 Use <<provisioning/auto-discovery.adoc#ga-provisioning-auto-discovery-detectors, auto discovery with detectors>> in addition to the ICMP IP address ping to specify services that you need to detect.
-xref:reference:provisioning/handlers/introduction.adoc#import-handlers[Import handlers] let you to further control provisioning.
+Use xref:reference:provisioning/handlers/introduction.adoc#import-handlers[import handlers] to further control provisioning.
```

Comment:
Import handlers are for getting nodes from different types of sources, typically those that aren't already xml formatted. Policies do more to control what is imported.

Reply:
Hmmm... okay. Does this look like a better description, then?

```suggestion
Use xref:reference:provisioning/handlers/introduction.adoc#import-handlers[import handlers] to provision nodes from other sources, including sources that may not be in XML format.
```

Reply:
I'm good with that wording.